plugins/tokens/plugin.go

```diff
@@ -104,9 +113,12 @@ func EndBlocker(ctx sdk.Context, timelockKeeper timelock.Keeper, swapKeeper swap
 			SwapID: swapID,
 		})
 		if !result.IsOK() {
-			logger.Error("Refund error", "swapId", swapID, "result", fmt.Sprintf("%+v", result))
+			logger.Error("failed to refund swap", "swapId", swapID, "result", fmt.Sprintf("%+v", result))
 			continue
 		}
+
+		logger.Info("succeed to refund swap", "swapId", swapID, "swap", fmt.Sprintf("%+v", swapItem))
+		swapKeeper.SetLatestProcessedRefundSwapKey(ctx, swapIterator.Key())
```

Comment:
Can we record the key and update the lasted proceed key outside the loop?

Reply:
if outside the loop, you will never know the key.  will this be heavy in `Endblock`?

Reply:
I've checked the codes, and found that `swap` would be gc in the breathe block, so I think this change is unnecessary


https://github.com/bnb-chain/node/blob/13bb81c70a49e55e296ed2b59c680ff9a85db8f5/plugins/tokens/plugin.go#L86C9-L86C9